auto/deps_pro-ha.yml

```diff
@@ -50,8 +50,16 @@ services:
     profiles: ["all", "master-datacenter"]
     container_name: restcountries
     image: bigpapoo/restcountries:1.0
-      
+
   trevorblades:
     profiles: ["all", "master-datacenter"]
     container_name: trevorblades
     image: mangomm/trevorblades-countries
+
+  webhook:
+    profiles: ["all", "master-datacenter"]
+    container_name: webhook-server
+    build:
+      context: ./webhook-server
+    ports:
+      - "9003:8080"
```

Comment:
Why not just keep it on 9003? In addition, do we need to expose any of the ports at all, due to shared docker networking?

Reply:
we can add 9003.
but we need to expose 9003 for local dev